### DISCUSSION TITLE
Citations / Zotero

### DISCUSSION THREAD

**Discussion by VEZY** [fonsp/Pluto.jl#1411] [Pluto development]
Hello, I'm wondering if there's some discussion about an integration of bibtex citations in the form @author_id in Markdown that would be rendered properly in the document (e.g. Author et al. (year)) ? And if so, it would be awesome to integrate with Zotero too. I know there is a javascript implementation for VSCode so maybe it could be used as a starting point ? https://github.com/mblode/vscode-zotero

It would be really awesome to write scientific reports with Pluto.

**Comment by saf-dmitry** [comment]
For now we can use Markdown links to Zotero, like this:

```markdown
[(Knuth 1976)](zotero://select/items/@knuth1976)
```

But a proper Zotero integration, like the [JupyterLab citation manager](https://github.com/krassowski/jupyterlab-citation-manager), would indeed be awesome.
